### DISCUSSION TITLE
Cheat request: Freeze the date

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#23012] [Feature requests]
Just like freezing the weather is useful, freezing the date can be useful too. If it is feasible to implement this I would love to see it added, especially now that the cheat menu has been revamped as now there's plenty of space for it.

**Comment by fidwell** [comment]
I know this isn't exactly what you asked for, but I created a plugin to freeze the date. Every time the date changes, it simply sets the date back by one day.

However it doesn't prevent things like research, salaries, loan interest, breakdowns, awards, etc. So I don't know how useful it will be for you.

https://gist.github.com/fidwell/29745676ed94cde24705e7860ee329e4
